CHANGELOG.md

```diff
@@ -32,7 +32,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added runtime option to auto spawn the ASpatialMetricsDisplay for remote debugging of server metrics - bEnableMetricsDisplay.
 - Added runtime option to batch spatial position updates to runtime.
 - Added runtime option to pack unreliable RPCs from the same frame in a single component update to save bandwidth - bPackUnreliableRPCs.
-- Enabled new Development Authentication Flow
+- Enabled new Development Authentication Flow.
+- An entity is now created for each server worker.
```

Comment:
```suggestion
- Added new ("worker") entities which are created for each server worker in a deployment to they correctly receive interest in the global state manager.
```

Reply:
Don't think you need the () around "worker"